README.md

```diff
@@ -1 +1,34 @@
-# distributed-task-scheduler
\ No newline at end of file
+# distributed-task-scheduler
+
+## Building and Running Applications
+
+This project contains multiple applications managed using Go workspaces. Below are the instructions to build and run each application.
+
+### Prerequisites
+
+- Ensure you have Go version 1.18 or later installed.
+- Clone the repository and navigate into the project's root directory.
+
+### Building Applications
+
+```bash
+#build worker
+cd worker
+go build -o worker
```

Comment:
I just want to add an idea. What do you think about creating a Makefile in the root of the project that will define two custom commands, for example they could be "make run worker" and "make run mts"?

This way the commands are easier to write and both can be done from the root.

Reply:
i would that as well but not in the scope of this ticket, make a new one for it